tds/src/main/java/thredds/server/wfs/WFSGetFeatureWriter.java

```diff
@@ -48,17 +75,14 @@ public void writeMembers() {
 					+ "</gml:boundedBy>"
 					
 					// Write Geometry Information
-					+ "<tds:hru_soil_moist gml:id=\"hru_soil_moist.1\">"
-					+ "<tds:catchments_geometry_container>"
-					+ "<gml:Point srsName=\"http://www.opengis.net/gml/srs/epsg.xml@900913\" srsDimension=\"2\">"
-					+ "<gml:pos>50.0 50.0</gml:pos>"
-					+ "</gml:Point>"
+					+ "<" + WFSController.TDSNAMESPACE + ":hru_soil_moist gml:id=\"hru_soil_moist.1\">"
+					+ "<" + WFSController.TDSNAMESPACE + ":catchments_geometry_container>"

```

Comment:
Good call to create and use this namespace constant!

Reply:
Thanks! Yeah allow allows for that flexibility to change in the future.